### ISSUE TITLE
border on hover does not work as expected when a source repo link is added.

### ISSUE THREAD

**Issue by cderv** [apreshill/quillt#12]
I may have miss something in #6

**Comment by apreshill** [comment]
Also a pkgdown issue: https://github.com/r-lib/pkgdown/issues/1519

Does dev pkgdown fix the issue? https://github.com/r-lib/pkgdown/pull/1521

**Comment by cderv** [comment]
Dev pkgdown fix the issue of not having the hover, but maybe it creates another one. See current https://pkgs.rstudio.com/bookdown/articles/articles/examples.html

**Comment by apreshill** [comment]
Oh yikes!

**Comment by cderv** [comment]
No it isn't **pkgdown** - the HTML source is ok - it seems th tweak I made are not correct 😞 Still looking.
Maybe I need to tweak the hover thing. Can you point me where is the piece of code doing this in the template ?

**Comment by apreshill** [comment]
This is just CSS:
https://github.com/apreshill/quillt/blob/2c58047aa3ef987c0659d06bd2244927c8d3aa4d/inst/pkgdown/assets/rmd.css#L415-L419

**Comment by apreshill** [comment]
Looks like the issue is that `thumbnail` as a class is being applied to the `<div class="caption">` elements

**Comment by apreshill** [comment]
From distill:
```
<a class="thumbnail" title="Piping Hot Data" href="https://www.pipinghotdata.com/">
  <img src="images/piping-hot-data.png">
  <div class="caption">Piping Hot Data</div>
</a>
```

**Comment by cderv** [comment]
Yes this is the issue. But I don't yet know why. I will check the HTML I am writing for source, I must have done a mistake there.

**Comment by cderv** [comment]
Oh I know. Rookie mistake. I made the source a link inside a link 🤕 not good.

**Comment by cderv** [comment]
Closed by #13
